package.json

```diff
@@ -1,38 +1,60 @@
 {
   "version": "1.0.0",
   "scripts": {
-    "test": "node_modules/.bin/lerna run test",
-    "lint": "node_modules/.bin/lerna run lint && node_modules/.bin/eslint './gulp/**/*.{js,jsx}' --fix",
-    "doc": "node_modules/.bin/lerna run doc",
-    "audit": "node_modules/.bin/lerna run audit",
-    "build": "node_modules/.bin/lerna run build",
-    "postinstall": "node_modules/.bin/lerna bootstrap --no-ci && node_modules/.bin/lerna run build"
+    "test": "lerna run test",
+    "lint": "lerna run lint && node_modules/.bin/eslint './gulp/**/*.{js,jsx}' --fix",
+    "doc": "lerna run doc",
+    "audit": "lerna run audit",
+    "postinstall": "lerna bootstrap --no-ci && lerna run build",
```

Comment:
There is no need for `lerna run build`, since it is part of `lerna boostrap` script

Reply:
Part `lerna run build` removed.